tkupchyn/homework_08/every_person.py

```diff
@@ -0,0 +1,48 @@
+# Details:  https://py.checkio.org/mission/every-person-is-unique/share/1c6fd4b663f7e40fa15b699f17efab40/
+
+
+from datetime import datetime, date
+
+
+class Person:
+    pronoun = {'male': 'He is a',
```

Comment:
if you put it all caps, you can highlight that this data is read only

PRONOUN = ...